cmd/testrunner/cmd/runtemplate/run_template.go

```diff
@@ -162,6 +164,7 @@ func init() {
 	runCmd.Flags().StringVar(&elasticSearchConfigName, "es-config-name", "sap_internal", "The elasticsearch secret-server config name.")
 	runCmd.Flags().StringVar(&s3Endpoint, "s3-endpoint", os.Getenv("S3_ENDPOINT"), "S3 endpoint of the testmachinery cluster.")
 	runCmd.Flags().StringVar(&concourseOnErrorDir, "concourse-onError-dir", os.Getenv("ON_ERROR_DIR"), "On error dir which is used by Concourse.")
+	runCmd.Flags().BoolVar(&updateIngest, "update-ingest", true, "Update the status of a ingested testrun.")
```

Comment:
Maybe that's more clear?
```suggestion
	runCmd.Flags().BoolVar(&updateIngest, "update-ingest", true, "Update the ingestion result (success/failure) of a testrun.")
```

Reply:
Maybe we should rename it completely to a more general term as maybe there are more status updates in the future?
How about `update-status`?

Reply:
Good idea. On a second thought: is there a reason why to not update the status? In which cases would one want to have the status not updated? So maybe we should always update and remove that option completely?

Reply:
yes true I could remove this flag and apply it as soon as the ingest function successfully finishes.
My thought was to add this possibility for later commands (get results of an already run testrun) . But yes it is really not needed

Reply:
Removed this flag